### ISSUE TITLE
G-RX8_ACCESS_V2.1.0 - can not detect  MLVSS or GPS sensors on SmartPort

### ISSUE THREAD

**Issue by NickMilton** [FrSkyRC/Firmware-Test#57]
Following these updates....
(1) Taranis 2019SE+ IRSM module from R1.1.3 to ISRM-S-X9DP2019_2.1.0B.frsk  
(2) Taranis 2019SE+ operating system from OpenTx R2.3.4 to opentx-x9d+2019-lua-noheli-ppmus-en-2.3.7
(2) G-RX8 Rx (with vario) Rx from R1.1.3 to G-RX8_ACCESS_2.1.0.frsk  

...the Taranis lost the ability to detect any type of sensor on the SmartPort. Wiring checked. Port pins checked. Power checked. No other changes made so I assume it must be a firmware issue in the G_RX8 rendering the SmartPort inoperative!

For context the telemetry from the G-RX8 itself (eg altimeter, RSSI etc) seems to be working correctly. Before the updates it had been working well for many months. (FCC not EU)

Any thoughts?.

**Comment by StefanKellerAC** [comment]
with a horus X10s and the upgrade module it works with
G-RX8_ACCESS_2.1.0_20200519.frsk and openXsensor GPS
did you delete and than discover all sensors new?

**Comment by NickMilton** [comment]
Thanks StefanKellerAC

Yes - several times to no avail. This is very odd because my other models with old X8R recievers running ACCST work fine and the sensors are easily discovered..

**Comment by StefanKellerAC** [comment]
did you go back to ACCST to test and than to ACCESS again?

**Comment by crazybiker84** [comment]
Hello Nick,
Did you re-register and bind the G-RX8 receiver after the update? I had the same problem on my Horus X-10s express. After I registered the receivers again, mlvss and GPS were recognized again.

**Comment by NickMilton** [comment]
> did you go back to ACCST to test and than to ACCESS again?

Thanks StefanKellerAC


No. This model has always been using ACCESS..

**Comment by NickMilton** [comment]
> Hello Nick,
> Did you re-register and bind the G-RX8 receiver after the update? I had the same problem on my Horus X-10s express. After I registered the receivers again, mlvss and GPS were recognized again.

Thanks for this insight...

No. I only rebound. I did not re-register! Good suggestion. I'll try this

**Comment by StefanKellerAC** [comment]
ok, but that would kill the advantage of ACCESS not to use the bind button anymore after registering once...

**Comment by crazybiker84** [comment]
> 
> 
> ok, but that would kill the advantage of ACCESS not to use the bind button anymore after registering once...

That's right, but after a firmware update it is imperative to re-register and bind the receiver.

**Comment by mpjf01** [comment]
@NickMilton Please advise if re-registering solved your problem or not.

**Comment by NickMilton** [comment]
> @NickMilton Please advise if re-registering solved your problem or not.

Being in Australia it is morning and I am at work. Will do this evening. Cheers

**Comment by NickMilton** [comment]
Resolution found!

Thanks to crazybiker84 I fully re-registered before rebinding. Then all the SPort sensors became visible on the Taranis. Of course in OpenTX Companion I then had to:

(1) remap all four telemetry screen displays
(2) redo the binding when I realised the Rx had defaulted to SBus mode - LOL
(3) check all logical switches and special functions - which were no longer all mapped to the correct telemetry tags 

Outcome - success!
